tomviz/modules/ModuleClip.cxx

```diff
@@ -587,6 +652,52 @@ void ModuleClip::onPlaneChanged(double* point)
   emit clipFilterUpdated(m_clippingPlane, false);
 }
 
+void ModuleClip::onInvertPlaneChanged() {
+  double normal[3];
+  m_widget->GetNormal(normal);
+  for (auto i = 0; i < 3; ++i) {
+    normal[i] *= -1;
+  }
+  m_widget->SetNormal(normal);
+  m_clippingPlane->SetNormal(normal);
+  onPropertyChanged();
+}
+
+void ModuleClip::onUpdateColor(const QColor& color) {
```

Comment:
These should probably all just be caught by clang-format, but opening braces get their own line for function level.